docs/schemas/GainCal.yml

```diff
@@ -87,30 +87,23 @@ inputs:
     doc: >-
       Debugging. If debuglevel==1, then a file debug.h5 is created containing all iterands. This file will be very large; you can use it to check the convergence speed etc `.`
   sourcedb:
-    type: 
     doc: >-
       Same as in :ref:`Predict` step `.`
   sources:
-    type: 
```

Comment:
This is a bug in the docs themselves, but the type in here should correspond to the type in the Predict step (in this case, `string?`. See https://github.com/lofar-astron/DP3/blob/master/docs/schemas/Predict.yml